### PR TITLE
Remove left-overs for building client bindings

### PR DESCRIPTION
**Pull request by lubosmj**
[noissue]

### CODE REVIEW

**Review by ggainey — DISMISSED** [review]

File: templates/github/.github/workflows/nightly.yml.j2 [hunk 1]
```diff
@@ -136,7 +136,7 @@ jobs:
 
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
-      {{ run_script(name="Install Python client", file="install_python_client.sh", withenv=False) | indent(6) }}
+      {{ run_script(name="Build Python bindings packages", file="build_python_client.sh", withenv=False) | indent(6) }}
```
Comment (mdellweg) [inline templates/github/.github/workflows/nightly.yml.j2:139]: I'm don't think we should still do this here.
This has moved to the build step. Either we do a download_artifact here or we let script.sh generate and install new ones.

File: templates/github/.github/workflows/publish.yml.j2 [hunk 2]
```diff
@@ -56,10 +56,10 @@ jobs:
 
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
-      {{ run_script(name="Install Python client", file="install_python_client.sh", withenv=False) | indent(6) }}
+      {{ run_script(name="Build Python bindings packages", file="build_python_client.sh", withenv=False) | indent(6) }}
 
       {%- if deploy_client_to_rubygems %}
-      {{ run_script(name="Install Ruby client", file="install_ruby_client.sh", withenv=False) | indent(6) }}
+      {{ run_script(name="Build Ruby bindings packages", file="build_ruby_client.sh", withenv=False) | indent(6) }}
```
Comment (mdellweg) [inline templates/github/.github/workflows/publish.yml.j2:62]: Same here. Actually the whole build-bindings-docs should have been done by build already. Can you completely remove that job from the workflow?

Comment (lubosmj) [reply templates/github/.github/workflows/publish.yml.j2:62]: Sure, let me try that.

File: templates/github/.github/workflows/publish.yml.j2 [hunk 3]
```diff
@@ -56,12 +56,6 @@ jobs:
 
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
-      {{ run_script(name="Install Python client", file="install_python_client.sh", withenv=False) | indent(6) }}
-
-      {%- if deploy_client_to_rubygems %}
-      {{ run_script(name="Install Ruby client", file="install_ruby_client.sh", withenv=False) | indent(6) }}
-      {%- endif %}
-
```
Comment (mdellweg) [inline templates/github/.github/workflows/publish.yml.j2:64]: I think, you can/should remove the whole "build-bindings-docs" job from the workflow.

Comment (lubosmj) [reply templates/github/.github/workflows/publish.yml.j2:64]: I, hopefully, removed the whole section. I need your expertise on verifying that.

**Review by mdellweg — APPROVED** [review]
Yes, I think, that is it.
Needs to be tested obviously by releasing a plugin.

**Comment by lubosmj** [comment]
Let's give it a try! 😆

**Pull request merged**